src/templates/work.development.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="Beenest" content="Beenest Platform">
   <title>Beenest</title>
-  <link href="https://fonts.googleapis.com/css?family=Roboto:100,400,500,700,900" rel="stylesheet">
```

Comment:
do we no longer need the font? or is it embedded in the css file/

Reply:
embedded